evap/rewards/tests/test_views.py

```diff
@@ -80,10 +80,20 @@ def test_redeem_points_for_expired_event(self):
         self.assertEqual(5, reward_points_of_user(self.student))
 
     def test_invalid_post_parameters(self):
-        self.app.post(self.url, params={"points-asd": 2}, user=self.student, status=400)
-        self.app.post(self.url, params={"points-": 2}, user=self.student, status=400)
-        self.app.post(self.url, params={f"points-{self.event1.pk}": ""}, user=self.student, status=400)
-        self.app.post(self.url, params={f"points-{self.event1.pk}": "asd"}, user=self.student, status=400)
+        point_params = {
+            "previous_reward_points": reward_points_of_user(self.student),
+            "previous_redeemed_points": redeemed_points_of_user(self.student),
+        }
+        self.app.post(self.url, params={"points-asd": 2, **point_params}, user=self.student, status=400)
+        self.app.post(self.url, params={"points-": 2, **point_params}, user=self.student, status=400)
+        self.app.post(self.url, params={f"points-{self.event1.pk}": "", **point_params}, user=self.student, status=400)
+        self.app.post(
+            self.url, params={f"points-{self.event1.pk}": "asd", **point_params}, user=self.student, status=400
+        )
+        # redemption without point parameters
+        self.app.post(self.url, params={f"points-{self.event1.pk}": 1}, user=self.student, status=400)
+        point_params["previous_reward_points"] = 0
+        self.app.post(self.url, params={f"points-{self.event1.pk}": 1, **point_params}, user=self.student, status=409)
```

Comment:
Personally, I would put this in a new test that does what we'd expect users to do:

* Load the page, store it in a variable.
* Load the page again, extract the form, submit it to redeem a point.
* Use the stored page to submit the form again, redeeming another point.
* Assert that 409 is returned.